cli/src/build.ts

```diff
@@ -58,14 +58,14 @@ export function setupWebsocket() {
         require("websocket-polyfill")
         // @ts-ignore
         global.Blob = require("buffer").Blob
-        global.WebSocket.prototype.send = function (this: any, data: any) {
+        global.WebSocket.prototype.send = function(this: any, data: any) {
```

Comment:
what kind of setting do you have in prettier? we should not have this kind of syntax formatting creeping in.